test/k8sT/manifests/externalIPs/README.md

```diff
@@ -62,4 +62,4 @@ that run on `k8s1` and `k8s2`:
 sudo service kube-proxy stop && sudo su -c 'iptables-save | grep -v KUBE | iptables-restore'
 ```
 
-and start cilium with `--enable-node-port --device=enp0s9`
+and start cilium with `--enable-node-port --device=enp0s10`
```

Comment:
nit: what happened to `enp0s9`? Can't `enp0s10` be used for `ip-masq` tests instead?

Reply:
For the multi-dev NodePort tests, I've introduced an additional private network (reachable via `enp0s9`), so the previous `enp0s9` was renamed to `enp0s10`.